docs/getting-started-guides/kops.md

```diff
@@ -0,0 +1,152 @@
+from clients.
+
+kops has a strong opinion on the cluster name: it should be a valid DNS name.  By doing so you will
+no longer get your clusters confused, you can share clusters with your colleagues unambigiously,
+and you can reach them without relying on remembering an IP address. 
+
+You can, and probably should, use subdomains to divide your clusters.  As our example we will use
+`useast1.dev.example.com`.  The API server endpoint will then be `api.useast1.dev.example.com`.
+
+A Route53 hosted zone can serve subdomains.  Your hosted zone could be `useast1.dev.example.com`,
+but also `dev.example.com` or even `example.com`.  kops works with any of these, so typically
+you choose for organization reasons (e.g. you are allowed to create records under `dev.example.com`,
+but not under `example.com`).
+
+Let's assume you're using `dev.example.com` as your hosted zone.  You create that hosted zone using
+the [normal process](http://docs.aws.amazon.com/Route53/latest/DeveloperGuide/CreatingNewSubdomain.html),
+and then you set up your NS records in the parent domain (here, you would create NS records in `example.com`
+for `dev`), or at your domain registrar if it is a root domain name (e.g. `example.com` would need to be configured
+where you bought `example.com`).
+
+This step is easy to mess up (it is the #1 cause of problems!)  You can double-check that
+your cluster is configured correctly if you have the dig tool by running:
+
+`dig NS dev.example.com`
+
+You should see the 4 NS records that Route53 assigned your hosted zone.
+
+### (3/5) Create an S3 bucket to store your clusters state
+
+kops lets you manage your clusters even after installation.  To do this, it must keep track of the clusters
+that you have created, along with their configuration, the keys they are using etc.  This information is stored
+in an S3 bucket.  S3 permissions are used to control access to the bucket.
+
+Multiple clusters can use the same S3 bucket, and you can share an S3 bucket between your colleagues that
+administer the same clusters - this is much easier than passing around kubecfg files.
+
+So typically you have one S3 bucket for each ops team (and often the name will correspond
+to the name of the hosted zone above!)
+
+In our example, we chose `dev.example.com` as our hosted zone, so let's pick `clusters.dev.example.com` as
+the S3 bucket name.
+
+* Export `AWS_PROFILE` (if you need to select a profile for the AWS CLI to work)
+
+* Create the S3 bucket using `aws s3 mb s3://clusters.dev.example.com`
+
+* You can `export KOPS_STATE_STORE=s3://clusters.dev.example.com` and then kops will use this location by default. 
+   We suggest putting this in your bash profile or similar. 
+
+
+### (4/5) Build your cluster configuration
+
+Run "kops create cluster" to create your cluster configuration:
+
+`kops create cluster --zones=us-east-1c useast1.dev.example.com`
```

Comment:
It is confusing that this doesn't actually start VMs.  Might want to clarify that this is just configuration and doesn't actually start machines.